src/krux/pages/home_pages/wallet_descriptor.py

```diff
@@ -39,41 +39,41 @@ def __init__(self, ctx):
         self.ctx = ctx
 
     def wallet(self):
-        """Handler for the 'wallet' menu item"""
+        """Handler for the 'wallet descriptor' menu item"""
+        title = t("Wallet output descriptor")
         self.ctx.display.clear()
         if not self.ctx.wallet.is_loaded():
             text = t("Wallet output descriptor not found.")
-            if not self.ctx.wallet.is_multisig():
-                text += " " + t("It is optional for single-sig.")
             self.ctx.display.draw_centered_text(text)
             if self.prompt(t("Load one?"), BOTTOM_PROMPT_LINE):
-                return self._load_wallet()
+                self._load_wallet()
         else:
             self.display_wallet(self.ctx.wallet)
             wallet_data, qr_format = self.ctx.wallet.wallet_qr()
-            title = t("Wallet output descriptor")
             from ..utils import Utils
 
             utils = Utils(self.ctx)
             utils.print_standard_qr(wallet_data, qr_format, title)
 
-            # Try to save the Wallet output descriptor on the SD card
-            if self.has_sd_card():
-                from ..file_operations import SaveFile
-
-                save_page = SaveFile(self.ctx)
-                save_page.save_file(
-                    self.ctx.wallet.descriptor.to_string(),
-                    self.ctx.wallet.label,
-                    self.ctx.wallet.label,
-                    title + ":",
-                    DESCRIPTOR_FILE_EXTENSION,
-                    save_as_binary=False,
-                )
+        # Try to save the Wallet output descriptor on the SD card
```

Comment:
removing this from the else clause introduced a bug alongside with the other change on the line 49 ` self._load_wallet()`